packages/cli/src/commands/governance/approve.ts

```diff
@@ -11,37 +11,47 @@ export default class Approve extends BaseCommand {
     ...BaseCommand.flags,
     proposalID: flags.string({ required: true, description: 'UUID of proposal to approve' }),
     from: Flags.address({ required: true, description: "Approver's address" }),
+    useMultiSig: flags.boolean({
```

Comment:
The "from" flag is unnecessary if "useMultiSig" is not set, right?

Reply:
i thought the point was you wanted users to use this without multisig.  in which case there is still a an address the request is coming from (which may be the multisig address or non-smart contract account). there would always be a from, but useMultiSig would indicate that the caller wants the request they are making to go through the multisig

Reply:
i don't want these to be mutually exclusive.  there will always be a from address needed.  we'll need to confirm the caller is a multisig signatory to perform the action

Reply:
Yeah I think my point was just that if they're not calling it via a multisig we can assume they're the approver and use that as the "from". Don't feel strongly